pages/agent/v3/upgrading.md.erb

```diff
@@ -24,17 +24,17 @@ Deprecated:
 
 * Built-in [Docker and Docker Compose support](/docs/tutorials/docker-containerized-builds) has been deprecated. The same functionality is available from the dedicated plugins: [docker-compose](https://github.com/buildkite-plugins/docker-compose-buildkite-plugin) and [docker](https://github.com/buildkite-plugins/docker-buildkite-plugin).
 
-## Bootstrap customizations
+### Bootstrap customizations
 
 If you customized your `bootstrap.sh` file, you will need to move the changes to [hooks](hooks), or update your bootstrap.sh to call `buildkite-agent bootstrap`.
 
-## Docker and Docker Compose support
+### Docker and Docker Compose support
 
 In v2 we supported a variety of environment variables like `BUILDKITE_DOCKER_COMPOSE_CONTAINER` and `BUILDKITE_DOCKER`. These are deprecated in favour of the [docker-compose](https://github.com/buildkite-plugins/docker-compose-buildkite-plugin) and [docker](https://github.com/buildkite-plugins/docker-buildkite-plugin) pipeline plugin.
```

Comment:
I think this section is only important for folks upgrading from v2 - maybe we can tuck it away in the `Upgrading from a 2.0 agent` section at the bottom?

Reply:
@yob wanna take a look at the updated results? If you're happy with that all I am.